src/main/java/com/openmrs/migrator/MigratorApplication.java

```diff
@@ -35,7 +35,8 @@
           "config/",
           "pdiresources/",
           "pdiresources/transformations/",
-          "pdiresources/jobs/");
+          "pdiresources/jobs/",
+          "plugins/");
```

Comment:
a lot has changed in this file, pull from develop please! these changes are now meant to go into Migrator.java

Reply:
:+1: